pkg/apis/istio/v1beta1/meshgateway_types.go

```diff
@@ -64,6 +65,7 @@ type MeshGatewayStatus struct {
 // +kubebuilder:printcolumn:name="Type",type="string",JSONPath=".spec.type",description="Type of the gateway"
 // +kubebuilder:printcolumn:name="Service Type",type="string",JSONPath=".spec.serviceType",description="Type of the service"
 // +kubebuilder:printcolumn:name="Status",type="string",JSONPath=".status.status",description="Status of the resource"
+// +kubebuilder:printcolumn:name="Gateways",type="string",JSONPath=".status.GatewayAddress",description="Ingress gateways of the resource"
```

Comment:
Thinking about better naming, something like this?
```suggestion
// +kubebuilder:printcolumn:name="Addresses",type="string",JSONPath=".status.GatewayAddress",description="Addresses of the Gateway resource"
```